docs/examples/arrow_example.ipynb

```diff
@@ -212,47 +207,60 @@
        "u_rated: [10500,10500,10500]"
       ]
      },
-     "execution_count": 19,
+     "execution_count": 4,
      "metadata": {},
      "output_type": "execute_result"
     }
    ],
    "source": [
-    "nodes_dict = {\"id\": [1, 2, 3], \"u_rated\": [10500.0, 10500.0, 10500.0]}\n",
-    "\n",
-    "\n",
-    "lines_dict = {\n",
-    "    \"id\": [4, 5],\n",
-    "    \"from_node\": [1, 2],\n",
-    "    \"to_node\": [2, 3],\n",
-    "    \"from_status\": [1, 1],\n",
-    "    \"to_status\": [1, 1],\n",
-    "    \"r1\": [0.11, 0.15],\n",
-    "    \"x1\": [0.12, 0.16],\n",
-    "    \"c1\": [4.1e-05, 5.4e-05],\n",
-    "    \"tan1\": [0.1, 0.1],\n",
-    "    \"r0\": [0.01, 0.05],\n",
-    "    \"x0\": [0.22, 0.06],\n",
-    "    \"c0\": [4.1e-05, 5.4e-05],\n",
-    "    \"tan0\": [0.4, 0.1],\n",
-    "}\n",
-    "\n",
-    "sources_dict = {\"id\": [6], \"node\": [1], \"status\": [1], \"u_ref\": [1.0]}\n",
+    "nodes_schema = pgm_schema(DatasetType.input, ComponentType.node)\n",
```

Comment:
```suggestion
    "# create the individual columns with the correct data type\n",
    "nodes_schema = pgm_schema(DatasetType.input, ComponentType.node)\n",
```

Reply:
these were not included in the latest run of readthedocs, it would appear. is that correct?